services/port_manager/src/main/java/com/futurewei/alcor/portmanager/util/RestParameterValidator.java

```diff
@@ -138,9 +138,9 @@ public static void checkRouterSubnetUpdateInfo(RouterUpdateInfo routerUpdateInfo
             throw new SubnetIdInvalid();
         }
 
-        RouterUpdateInfo.OperationType operationType = routerUpdateInfo.getOperationType();
-        if (!RouterUpdateInfo.OperationType.ADD.equals(operationType) &&
-                !RouterUpdateInfo.OperationType.DELETE.equals(operationType)) {
+        String operationType = routerUpdateInfo.getOperationType();
+        if (!RouterUpdateInfo.OperationType.ADD.getType().equals(operationType) &&
```

Comment:
We would need a case-insensitive check.

Reply:
@chenpiaoping Check if this is a problem. If not, we can go ahead to merge it.

Reply:
I think it's okay.

Reply:
Fixed.